lib/meal_choice.rb

```diff
@@ -0,0 +1,10 @@
+# Define a method `meal_choice`
+# It takes 2 veggies and protein defaults to meat
+# It returns `meal` with a value of
+# "A plate of #{protein} with #{veg1} and #{veg2}"
+# and `puts` "What a nutritious meal!"
+
+def meal_choice(veg1, veg2, protein='meat')
+  puts "What a nutritious meal!"
+  meal = "A plate of #{protein} with #{veg1} and #{veg2}"
```

Comment:
i'm not sure implicit return is the clearest thing in the world yet to this audience. Have we covered it

1. prefer explicit return of `meal`
2. IF you want to show implicit return, then do not set `meal` b/c all you're getting back is the String anyway.

If you do want to use impl. return, put that in your pseudocode.

Reply:
@sgharms I think this is an area where we can remind them that either works, and maybe add alternatives to the solution (commented out)?